Library/Formula/silk.rb

```diff
@@ -17,8 +15,12 @@ def install
                           "--enable-ipv6",
                           "--enable-data-rootdir=#{var}/silk"
     system "make"
-    system "make install"
+    system "make", "install"
 
     (var+"silk").mkpath
   end
+
+  test do
+    system "#{bin}/silk_config", "--version"
```

Comment:
Knowing nothing about SiLK Flow files, I'd leave this for other (ideally package-native) contributors.